projects/igniteui-angular/src/lib/combo/combo-dropdown.component.ts

```diff
@@ -377,10 +390,16 @@ export class IgxComboDropDownComponent extends IgxDropDownBase implements OnDest
         this.verticalScrollContainer.getVerticalScroll().scrollTop = this._scrollPosition;
     }
 
+    public ngAfterViewInit() {
+        this._vScrollListener = this.verticalScrollHandler.bind(this);
+        this.verticalScrollContainer.getVerticalScroll().addEventListener('scroll', this._vScrollListener);
```

Comment:
Can you convert `this.verticalScrollHandler` to an arrow function and just use it directly for event binding (drop the extra ref `_vScrollListener`)